src/View/Components/ToggleTheme.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace TallStackUi\View\Components;
+
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Arr;
+use TallStackUi\Foundation\Attributes\SoftPersonalization;
+use TallStackUi\Foundation\Personalization\Contracts\Personalization;
+
+#[SoftPersonalization('toggleTheme')]
+class ToggleTheme extends BaseComponent implements Personalization
+{
+    public function __construct(
+        public ?bool $simple = false,
```

Comment:
Although this makes sense and is interesting, it doesn't have much to do with the component's name being "Toggle Theme" and only displaying an icon. So, I suggest renaming the component to "Theme Switch", and then it will make sense: default as a toggle with an option for display an icon instead of toggle.

Reply:
Adopting the idea of naming it as "Theme Switch", in the future we can offer a button format and a text slot, something generic and not just a toggle.